source/developers/sts-for-operator.rst

```diff
@@ -0,0 +1,186 @@
+.. _minio-sts-operator:
+
+===============================================
+Security Token Service (STS) for MinIO Operator
+===============================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+.. versionadded:: Operator v5.0.0
+
+   The MinIO Operator supports a set of API calls that allows an application to obtain STS credentials for a MinIO Tenant.
+
+Benefits of STS for MinIO Operator include:
+
+- :ref:`STS credentials <minio-security-token-service>` allow an application to access objects on a MinIO Tenant without the need to create credentials for the application on the tenant.
+
+- Allows applications to access objects in MinIO tenants using a Kubernetes-native authentication mechanism.
+  
+  Service Accounts or Service Account Tokens are a core concept of :kube-docs:`Role-Based Access Control (RBAC) <reference/access-authn-authz/rbac/>` :kube-docs:`authentication <reference/access-authn-authz/authentication/#service-account-tokens>` in Kubernetes.
+
+- Implementing STS for MinIO Operator allows you to utilize infrastructure as code principles and configuration by using the tenant custom resource definition (CRD) and a MinIO PolicyBinding CRD.
+
+.. important:: 
+
+   MinIO Operator launches with STS *disabled* by default.
+   To use STS with the Operator, you must first explicitly enable it.
+
+   The procedure on this page includes instructions to enable the STS API in the MinIO Operator.
+
+How STS Authorization Works in Kubernetes
+-----------------------------------------
+
+An application can use an ``AssumeRoleWithWebIdentity`` call including a Service Account's :abbr:`JWT (JSON Web Token)` to send a request for temporary credentials to the MinIO Operator.
+The JWT is typically mounted in the deployment from a well known location on the deployment, such as ``/var/run/secrets/kubernetes.io/serviceaccount/token``.
```

Comment:
From what Pedro said, it's more that

"Kubernetes mounts Service Accounts to pods specifying that SA using <whatever that field was>.
The Pod can access those SAs at a well known location such as <URL>"